pkg/envoy/accesslog_server.go

```diff
@@ -137,6 +139,7 @@ func (s *AccessLogServer) handleConn(ctx context.Context, conn *net.UnixConn) {
 			break
 		}
 		if flags&unix.MSG_TRUNC != 0 {
+			// If you see this message, consider increasing the buffer size of the access log server.
 			log.Warning("Envoy: Discarded truncated access log message")
```

Comment:
maybe worth to add the configured buffer size as a log field? IMO this could help to "link" the warning to the configurable buffer size (in addition to your comment that requires browsing the source code)

e.g. `WithField("accesslog_buffer_size", bufferSize)`

Reply:
Updated in https://github.com/cilium/cilium/commit/4b53f2e574c04c3ac52abe07afbd228db18c3978 - I removed the comment since it feels redundant with the log message